tests/test_api.py

```diff
@@ -655,9 +683,52 @@ def test_visualize_tangled_tree_layers(self, client, figure_type, data_model_jso
         assert response.status_code == 200
 
 
+@pytest.mark.schematic_api
+class TestValidationBenchmark():
+    @pytest.mark.parametrize('MockComponent_attribute', get_MockComponent_attribute())
+    def test_validation_performance(self, benchmark_data_model_jsonld, client, test_invalid_manifest, MockComponent_attribute ):
+        target_rows = 1000
+        endpoint_url = 'http://localhost:3001/v1/model/validate'
+
```

Comment:
Could you add some comments for the code block below? It seems like this part is changing the dataframe `test_invalid_manifest` for testing purpose?